pyxem/signals/diffraction2d.py

```diff
@@ -582,6 +584,61 @@ def remove_background(self, method, **kwargs):
 
         return bg_subtracted
 
+    def as_polar(self, dr=1.0, dt=None, jacobian=False, *args, **kwargs):
```

Comment:
1) is it worth setting `jacobian` to True here given that it's what the docstrings are encouraging
2) `.map()` doesn't take `*args`, so can we cut it.